Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -74,6 +74,8 @@ module.exports = class DoubleClickToEdit {
 
 			//Load settings
 			this.doubleClickToReplySetting = BdApi.loadData(config.info.id, "doubleClickToReplySetting") ?? false;
+			this.replyActionKey = BdApi.loadData(config.info.id, "replyActionKey") ?? "any";
+			this.editActionKey = BdApi.loadData(config.info.id, "editActionKey") ?? "any";
 			this.copyBeforeAction = BdApi.loadData(config.info.id, "copyBeforeAction") ?? false;
 			this.copyBeforeActionModifier = BdApi.loadData(config.info.id, "copyBeforeActionModifier") ?? "shift";
```

Comment:
```suggestion
			//Edit
			this.doubleClickToEditModifier = Data.load(config.info.id, "doubleClickToEditModifier") ?? false;
			this.editModifier = Data.load(config.info.id, "editModifier") ?? "shift";
			//Reply
			this.doubleClickToReply = Data.load(config.info.id, "doubleClickToReply") ?? false;
			this.doubleClickToReplyModifier = Data.load(config.info.id, "doubleClickToReplyModifier") ?? false;
			this.replyModifier = Data.load(config.info.id, "replyModifier") ?? "shift";
			//Copy
			this.doubleClickToCopy = Data.load(config.info.id, "doubleClickToCopy") ?? false;
			this.copyModifier = Data.load(config.info.id, "copyModifier") ?? "shift";
```
With this change a lot of the settings need to be redone and clarified.